docs: add a clear note that we do not strictly follow PSR/PER (#2394)
prettier/plugin-php: README.md

This plugin adds support for the PHP language to Prettier.

> [!NOTE]
>

> > This plugin has always used PSR / PER as guidance when making formatting decisions, but has never aimed to be fully PSR / PER compliant. The idea behind this plugin is to stay reasonably close to how Prettier for JS works.

Reviewer: ```suggestion
> This plugin uses PSR / PER as guidance when making formatting decisions, but does not aim to be fully PSR / PER compliant. The idea is to stay reasonably close to how Prettier for JS works.
```
